### PR TITLE
Create Hungarian language file

### PR DESCRIPTION
**Pull request by frodo19**
Create hungarian lang

### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by frodo19** [comment]
Create hungarian lang

**Comment by CastagnaIT** [comment]
Thank you for this new language,
now you can update with last changes

so you also have to add on the file addon.xml the language
on tag <language> line 38 in alphabetic order, so between "hr" and "it"

in the same file if you want you can translate the <description> and <disclaimer> tags

File: resources/language/resource.language.hu_hu/strings.po [hunk 1]
```diff
@@ -0,0 +1,977 @@
...
+msgid "(between 0 & 10)"
+msgstr "0 és 10 között"
+
+msgctxt "#30023"
+msgid "Expert"
+msgstr "Haladó"
+
+msgctxt "#30024"
+msgid "SSL verification"
+msgstr ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: missing translation

File: resources/language/resource.language.hu_hu/strings.po [hunk 2]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Enable HDR profiles"
+msgstr "HDR profil engedélyezése"
+
+msgctxt "#30099"
+msgid "Enable DolbyVision profiles"
+msgstr "DolbyVision profil engedélyezése"
+
+msgctxt "#30100"
+msgid "Results for \"{}\""
+msgstr "Találat \"{}\"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: missing one " at the end

File: resources/language/resource.language.hu_hu/strings.po [hunk 3]
```diff
@@ -0,0 +1,977 @@
...
+
+msgctxt "#30101"
+msgid "Netflix returned an unknown error."
+msgstr "A Netflix leállt ismeretlen hiba miatt"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: missing dot at end

File: resources/language/resource.language.hu_hu/strings.po [hunk 4]
```diff
@@ -0,0 +1,977 @@
...
+
+msgctxt "#30102"
+msgid "Netflix returned the following error:"
+msgstr "A Netflix leállt a következő hiba miatt:"
+
+msgctxt "#30103"
+msgid "Error details have been written to the Kodi logfile. If the error message does not provide guidance on how to resolve the issue or you continue to receive this error, please report it via GitHub. Please provide a full debug log with your error report (enable \"Debug Logging\" in Kodi settings)."
+msgstr "A hiba részleteit a Kodi a naplóba mentette. Ha a hibaüzenet nem ad útmutatást a probléma megoldására, vagy továbbra is jelentkezik a hiba, kérjük, jelentse a GitHubon keresztül. Adjon meg egy teljes hibakeresési naplót a hibajelentésével (engedélyezze a \ "Hibakeresés \" lehetőséget a Kodi beállításokban)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: there is a space to remove after \

Comment (CastagnaIT) [reply resources/language/resource.language.hu_hu/strings.po]: and missing the dot at end

File: resources/language/resource.language.hu_hu/strings.po [hunk 5]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Perform full sync now"
+msgstr "Teljes szinkronizálás indítása most"
+
+msgctxt "#30122"
+msgid "Sync My List to Kodi library"
+msgstr "Saját lista szinkronizálása a Kodi médiatárral"
+
+msgctxt "#30123"
+msgid "Do you really want to proceed?\nThis will delete all previously exported items and export all items from My List for the currently active profile.\nThis operation may take some time..."
+msgstr "Tényleg folytatni akarja? \ Ezzel törli az összes korábban exportált elemet, és exportálja az összes elemet a Saját listából az aktív profilhoz. \ Ez a művelet eltarthat egy ideig ..."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: the "\\" must be "\n"

File: resources/language/resource.language.hu_hu/strings.po [hunk 6]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Do you really want to remove this item?"
+msgstr "Valóban törlöd ezt az elemet?"
+
+msgctxt "#30125"
+msgid "Purge library"
+msgstr "Könyvtár tisztítása"
+
+msgctxt "#30126"
+msgid "Do you really want to purge your library?\nAll exported items will be deleted."
+msgstr "Valóban tisztítani szertnéd a könyvtárat? \ Az összes exportált elem törlődik."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: the "\\" must be "\n"

File: resources/language/resource.language.hu_hu/strings.po [hunk 7]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Netflix addon configuration"
+msgstr "Netflix kiegészítő beállítása"
+
+msgctxt "#30155"
+msgid "Do you have a Netflix Premium account?"
+msgstr "Van Netflix prémium hozzáférésed?"
+
+msgctxt "#30156"
+msgid "Does your device support the 4K standard?\r\nThe device must:\r\nBe Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr "Támogatja-e az eszközöd a 4K-szabványt? \ Az eszköznek: \ kell rendelkeznie Netflix tanúsítvánnyal, támogatnia kell az L1 Widevine biztonsági szintet, és rendelkeznie kell 4K HDCP 2.2 kijelzővel."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: the "\\" must be "\r\n"

File: resources/language/resource.language.hu_hu/strings.po [hunk 8]
```diff
@@ -0,0 +1,977 @@
...
+
+msgctxt "#30157"
+msgid "The InputStream Adaptive settings panel will now be opened, so you have to change this option \"Override HDCP status\" to ON"
+msgstr "Most megnyílik az InputStream Adaptive beállítások panele, tehát ezt a \ "HDCP állapot felülbírálása \" beállítást ON-ra kell állítani."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: the space after \ to be remove

File: resources/language/resource.language.hu_hu/strings.po [hunk 9]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Show trailers & more"
+msgstr "Bemutatók és egyebek mutatása"
+
+msgctxt "#30180"
+msgid "There has been a problem with login, it is possible that your account has not been confirmed or renewed."
+msgstr "Probléma merült fel a bejelentkezés során, lehetséges, hogy fiókját nem erősítették meg vagy nem újították meg"
+
+msgctxt "#30181"
+msgid "Disable subtitles if there are no forced subtitles streams (Works only if Kodi Player is set to \"Only forced\" subtitles)"
+msgstr "A feliratok letiltása, ha nincs kényszerített felirat (csak akkor működik, ha a Kodi  "Csak kényszerített" feliratra van állítva)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: missing escape character before the chars "  must be \\"

File: resources/language/resource.language.hu_hu/strings.po [hunk 10]
```diff
@@ -0,0 +1,977 @@
...
+msgid "TV show"
+msgstr "sorozat"
+
+msgctxt "#30191"
+msgid "Do you want to export NFO files for the {} added to 'My List'?"
+msgstr "NFO fájlok exportálása a saját listámhoz?"
+
+msgctxt "#30192"
+msgid "\r\nThese files are used by the provider information scrapers to integrate movies and tv shows info, useful with bonus episodes or director's cut"
+msgstr "Ezeket a fájlokat a szolgáltatói információs scrapperek használják a filmek és a sorozatok információinak integrálására, hasznosak a bónusz epizódok vagy a rendezői vágások során."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: missing \r\n at start

File: resources/language/resource.language.hu_hu/strings.po [hunk 11]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Show ESN in use"
+msgstr "ESN használat mutatása"
+
+msgctxt "#30217"
+msgid "Reset ESN"
+msgstr "ESN reset"
+
+msgctxt "#30218"
+msgid "Are you sure you want to reset the ESN?\r\nIf you confirm, wait until you receive the login message."
+msgstr "Biztosan vissza akarja állítani az ESN-t? \ Ha megerősíti, várjon, amíg megkapja a bejelentkezési üzenetet."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: also here \r\n

File: addon.xml [hunk 12]
```diff
@@ -35,7 +38,7 @@
       <screenshot>resources/media/screenshot-04.jpg</screenshot>
       <screenshot>resources/media/screenshot-05.jpg</screenshot>
     </assets>
-    <language>en de es he hr it nl pl pt sk sv</language>
+    <language>en de es he hu hr it nl pl pt sk sv</language>
```
Comment (CastagnaIT) [inline addon.xml]: position wrong, must be between "hr" and "it"

File: resources/language/resource.language.hu_hu/strings.po [hunk 13]
```diff
@@ -0,0 +1,977 @@
...
+msgid "Netflix addon configuration"
+msgstr "Netflix kiegészítő beállítása"
+
+msgctxt "#30155"
+msgid "Do you have a Netflix Premium account?"
+msgstr "Van Netflix prémium hozzáférésed?"
+
+msgctxt "#30156"
+msgid "Does your device support the 4K standard?\r\nThe device must:\r\nBe Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr "Támogatja-e az eszközöd a 4K-szabványt?\rAz eszköznek:\r\nKell rendelkeznie Netflix tanúsítvánnyal, támogatnia kell az L1 Widevine biztonsági szintet, és rendelkeznie kell 4K HDCP 2.2 kijelzővel."
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: \r is wrong \r\n

File: resources/language/resource.language.hu_hu/strings.po [hunk 14]
```diff
@@ -0,0 +1,977 @@
...
+
+msgctxt "#30231"
+msgid "Do you want to check update now?"
+msgstr "Akarsz most keresni frissítést?"
+
+msgctxt "#30232"
+msgid "Restrict by Maturity Level"
+msgstr "Kor szeint korlátozva"
+
+msgctxt "#30233"
```
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po]: from this line to the end of file, all is changed, compare with en file language

**Comment by CastagnaIT** [comment]
Thank you

**Comment by CastagnaIT** [comment]
@frodo19 
I made a manual check and i found out that you hadn't updated all previous ids...
I made the correction, but they will be to be translated (see last commit master branch)
Next time you have to use a file comparison software
to prevent fewer errors

**Pull request merged**